sql/expression/like_test.go

```diff
@@ -65,6 +65,7 @@ func TestLike(t *testing.T) {
 		{"a%b", "ab", true},
 		{"a%b", "a", false},
 		{"a_b", "ab", false},
+		{"aa:%", "AA:BB:CC:DD:EE:FF", true},
```

Comment:
this test could be simpler, why not
```
{"aa:%", "AA:BB", true},
```

Reply:
no reason.  Was using a valid MAC address format like what we see in the nautobot code.

Reply:
ahh, got it